src/components/FormList.js

```diff
@@ -0,0 +1,131 @@
+
+
+import React, {useState} from 'react';
+import {getTodaysDate} from "./Date";
+
+// import { useForm } from "react-hook-form";
+
+const FormList = (props) => {
```

Comment:
I think it would make sense to introduce a `dirty` state that indicates whether the user was trying to create the challenge. This way we can hide the validation messages while the form is being filled in, and only show them when the user clicks the button.